spinalcordtoolbox/scripts/sct_dmri_denoise_patch2self.py

```diff
@@ -132,7 +132,7 @@ def main(argv: Sequence[str]):
     nib.save(img_diff, output_file_name_diff)
 
     printv('\nDone! To view results, type:', verbose)
-    printv('fsleyes ' + file_to_denoise + ' ' + output_file_name_denoised + ' & \n',
+    printv('fsleyes ' + file_to_denoise + ' ' + output_file_name_denoised + ' ' + output_file_name_diff + ' & \n',
            verbose, 'info')
```

Comment:
Oh. Huh. We shouldn't really be using `printv` at all for this, actually! 

This kind of output is standardized via the [`display_viewer_syntax`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/6b1791b90d998e5a0ad3b1aedb3a6f29ac9a2981/spinalcordtoolbox/utils/shell.py#L43-L87) function (which auto-detects available viewers, and accounts for additional viewers beyond just `fsleyes`).

```suggestion
    display_viewer_syntax([file_to_denoise, output_file_name_denoised, output_file_name_diff], verbose=verbose)
```

Reply:
Ah, you've already done this one, too! :smile: